service/kas/kas.go

```diff
@@ -58,7 +58,7 @@ func NewRegistration() serviceregistry.Registration {
 		ServiceDesc: &kaspb.AccessService_ServiceDesc,
 		RegisterFunc: func(srp serviceregistry.RegistrationParams) (any, serviceregistry.HandlerServer) {
 			// FIXME msg="mismatched key access url" keyAccessURL=http://localhost:9000 kasURL=https://:9000
```

Comment:
@pflynn-virtru was there an issue to track this?

Reply:
I've filed #578. Feel free to comment there with more details, @jakedoublev or @pflynn-virtru